### DISCUSSION TITLE
Renovate does not use togithub redirect in some PR body text

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#25602] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Renovate's PR body text links to `github.com` instead of the `togithub.com` redirect.

[Minimal reproduction on HonkingGoose repository](https://github.com/HonkingGoose/25602)

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment, marked as answer]
I think it's ok to link directly to github.com so long as it doesn't call "backlink spam". In other words we want to avoid linking to issues, PRs or users, but if it's only a diff link then there's no problem as it doesn't notify or spam anyone

**Reply by HonkingGoose** [reply]
Oh okay! I thought we _always_ wanted to use the redirect link. 😄
